crates/autopilot/src/boundary/auction.rs

```diff
@@ -0,0 +1,14 @@
+use crate::domain;
+
+pub fn to_domain(auction: model::auction::Auction) -> domain::Auction {
```

Comment:
I'm still sceptical if we should use `model::auction::Auction` at all in the autopilot. Can we not directly create the `domain::Auction` type inside `solvable_orders::update` ?

Reply:
Yes, but as you mentioned I have to draw a line somewhere with regards to the scope of this PR. Refactoring of solvable orders cache and moving in to the `domain` completely could be done in a separate PR.